Mathlib/Algebra/GroupPower/NegOnePow.lean

```diff
@@ -1,7 +1,7 @@
 /-
 Copyright (c) 2023 Joël Riou. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Joël Riou, Johan Commelin
+Authors: Joël Riou, Johan Commelin, Mitchell Lee
```

Comment:
My understanding is that the authors in the copyright line currently used not as "everybody who contributed anything to the file", but rather as "people who contributed a lot - so if we have a question, we ask them".
Your lemmas are nice, but imho don't fundamentally change the file; so personally, I'd remove myself. (This is not to say your contribution were unimportant!)